bin/admin/install-solvcon.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+if [ $# -lt 1 ]
+then
+    echo "Usage : $0 <solvcon env configuration file>"
+    exit
+fi
+solvcon_env_file=$1
+git clone https://github.com/solvcon/solvcon.git
+
+# you never know the env is ready to use SOLVCON or not, so
+# enable the env anyway.
+source $solvcon_env_file
+# basic env
+export SCSRC=$HOME/solvcon
+# prepare the ssh configuration to test rpc features on the localhost
```

Comment:
I think SSH public key generation should be don't independent to SOLVCON installation.